### DISCUSSION TITLE
Disable JavaScript by default?

### DISCUSSION THREAD

**Discussion by r3k2** [Jelmerro/Vieb#523] [Q&A] [answered]
Hello is there a way to disable all javascript by default and allow only certain trusted sites with it?

Thanks

**Comment by Jelmerro** [comment, marked as answer]
You can customize where external JS (and other) resources are allowed with `resourcetypes`, see `:help resourcetypes` for details. Keep in mind that due to a technical limitation, inline JS will still run, but external resources can be controlled with this setting.
